src/shell/scripts/omp.ps1

```diff
@@ -1,249 +1,183 @@
-function global:Start-Utf8Process {
-    param(
-        [string] $FileName,
-        [string] $Arguments
-    )
+New-Module -Name "OhMyPosh" -ScriptBlock {
```

Comment:
Can we rename this to `oh-my-posh-init`? The standard module is called `oh-my-posh` so this aligns the names a bit. Other than that, I'm impressed. Amazing work (also validated it myself).

Reply:
💡 Just a suggestion: how about name it "oh-my-posh-core"? The dynamic module not only does the init work, but also persists in the PowerShell session and plays a "core" role in wrapping OMP calls.

Reply:
Perfect!